docs/sources/configuration/query-frontend.md

```diff
@@ -60,7 +60,7 @@ data:
 
     frontend:
       log_queries_longer_than: 5s
-      downstream_url: querier.<namespace>.svc.cluster.local:3100
+      downstream_url: http://querier.<namespace>.svc.cluster.local:3100
```

Comment:
Not that its a problem setting a scheme explicitly here but I wonder if you are facing any issue without setting the it. We do not face any issue when it is not set.

Reply:
Yes, I just tested it. Without http://, it gives the following error  regarding unsupported protocol scheme

```
level=warn ts=2021-01-04T14:37:38.35781553Z caller=logging.go:62 traceID=8c074c80cfcc1a msg="GET /ready (500) 156.572µs Response: \"unsupported protocol scheme \\\"loki.service.tr3.consul\\\"\\n\" 
```